shopware/paas-meta/6.6/root/.environment

```diff
@@ -0,0 +1 @@
+export APP_CACHE_DIR=$PLATFORM_APP_DIR/localCache
```

Comment:
is this a platform.sh specific file?

and why moved here?

Reply:
Yes `.environment` file is a specific P.sh config file, see https://docs.platform.sh/development/variables/set-variables.html#set-variables-via-script.

Initially `APP_CACHE_DIR` was configured in the `applications.yaml` file, but expansion doesn't work in env variables defined in the `applications.yaml`, and we needed to include $PLATFORM_APP_DIR so we had no other choice.